salt/modules/rh_ip.py

```diff
@@ -808,7 +722,7 @@ def _parse_settings_eth(opts, iface_type, enabled, iface):
         result["enable_ipv6"] = opts["enable_ipv6"]
 
     valid = _CONFIG_TRUE + _CONFIG_FALSE
-    for opt in [
+    for opt in (
```

Comment:
I'm curious why you are changing this from a list to tuple here but in other places are changing these kinds of checks from tuples to lists? Would it be better to standardize on tuples if they don't need to be mutable?

Reply:
I don't think any of my changes involved changing tuples to lists, did I miss anything?